lua/starfall/libs_sh/entities.lua

```diff
@@ -829,6 +829,50 @@ function ents_methods:getBonePosition(bone)
 	return vwrap(pos), awrap(ang)
 end
 
+--- Returns the angle manipulate angle of the bone (local to the entity)
+-- @shared
+-- @param bone Bone index. (def 0)
+-- @return Manipulate angle of the bone
+function ents_methods:getManipulateBoneAngles(bone)
+	local ent = getent(self)
+	if bone == nil then bone = 0 else checkluatype(bone, TYPE_NUMBER) end
+	local ang = ent:GetManipulateBoneAngles(bone)
+	return awrap(ang)
+end
+
+--- Returns the number manipulate jiggle of the bone
```

Comment:
Would be nice to specify that this function returns a number in the range 0..255